dbt_subprojects/hourly_spellbook/models/_sector/perpetual/projects/pika/optimism/pika_v4_optimism_perpetual_trades.sql

```diff
@@ -0,0 +1,171 @@
+{{ config(
+	
+	schema = 'pika_v4_optimism',
+	alias = 'perpetual_trades',
+	partition_by = ['block_month'],
+	materialized = 'incremental',
+	file_format = 'delta',
+	incremental_strategy = 'merge',
+	unique_key = ['block_date', 'blockchain', 'project', 'version', 'tx_hash', 'evt_index'],
+    post_hook='{{ expose_spells(\'["optimism"]\',
+                                "project",
+                                "pika",
+                                \'["msilb7", "drethereum", "rplust","princi"]\') }}'
```

Comment:
We can apply the new post_hook format here.
i.e.
https://github.com/duneanalytics/spellbook/blob/d09fa392c8e07e6443cf26c4e59c97f1c7e41e8d/dbt_subprojects/dex/models/_projects/lifi/lifi_trades.sql#L4